app/views/layouts/application.html.haml

```diff
@@ -34,6 +34,9 @@
     #{yield :analytics}
     ga('set', 'anonymizeIp', true);
     ga('send', 'pageview');
+  - if Rails.env == 'test'
+    :javascript
+      moj.log = function(){}; // Silence hiring banner
```

Comment:
Can someone please, please, please just kill the hiring banner once and for all?